### PR TITLE
ci: add ci check for gofmt

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

add ci check for gofmt

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@nic-chen would better request reviewers so they could see the notification.

File: .github/workflows/go-lint.yml [hunk 1]
```diff
@@ -10,9 +10,26 @@ on:
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13'
+      - name: check gofmt
+        working-directory: ./
+        run: |
+          diffs=`gofmt -l .`
```
Comment (starsz) [inline .github/workflows/go-lint.yml:30]: IMO, `gofmt -l -d .` would be better.

Comment (nic-chen) [reply .github/workflows/go-lint.yml:30]: OK

Comment (nic-chen) [reply .github/workflows/go-lint.yml:30]: oh, no, I try this, it display too much info, it's unnecessary to show the detail in the ci.
@starsz  you could try for it..

Comment (starsz) [reply .github/workflows/go-lint.yml:30]: `-d display diffs instead of rewriting files`, so that we can solve the `go fmt` problem.

**Review by starsz — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**